tests/integration/test_zero_copy_expand_macros/test.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+
+def get_macros():
+    return {
+        "shard": "01",
+        "replica": "1",
+    }
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.add_instance(
+            "node",
+            main_configs=["configs/storage_conf.xml"],
+            with_minio=True,
+            with_zookeeper=True,
+            macros=get_macros(),
+        )
+        cluster.start()
+
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_drop_after_fetch(started_cluster):
+    node = cluster.instances["node"]
+
+    node.query(
+        """
+        CREATE TABLE test_s3(c1 Int8, c2 Date)
+        ENGINE = ReplicatedMergeTree('/test/tables/{shard}/test_s3', '1')
+        PARTITION BY c2
+        ORDER BY c2
+        SETTINGS
+            storage_policy = 's3',
+            allow_remote_fs_zero_copy_replication=1,
+            remote_fs_zero_copy_zookeeper_path = '/clickhouse/zero_copy/{shard}/{replica}'
```

Comment:
By the way, path must be the same for different replicas in one shard, because Keeper is used here for synchronization between replicas. Don't think that need to make some check for it (`replica` is just a macros name), but better to make test only with `shard` to avoid confusing someone.